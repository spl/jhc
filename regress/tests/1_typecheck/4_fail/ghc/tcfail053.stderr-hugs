ERROR "tcfail053.hs":3 - Undefined type constructor "A"
