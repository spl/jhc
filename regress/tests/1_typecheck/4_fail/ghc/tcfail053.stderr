
tcfail053.hs:3:12: Not in scope: type constructor or class `A'
